CHANGELOG.en-US.md

```diff
@@ -54,7 +54,9 @@ timeline: true
   - 🆕 Tree support `onScroll` in virtual scroll. [#474](https://github.com/react-component/tree/pull/474)
   - 🐞 Fix Tree drop outside not clear indicator. [#478](https://github.com/react-component/tree/pull/478)
 - 💄 Add `@checkbox-border-radius` less variable. [#31360](https://github.com/ant-design/ant-design/pull/31360) [@Gherciu](https://github.com/Gherciu)
-- 🐞 Avatar add `crossOrigin` property to resolve cross origin issue. [#31273](https://github.com/ant-design/ant-design/pull/31273) [@Map1en](https://github.com/Map1en)
+- 🐞 Avatar
+  - add `crossOrigin` property to resolve cross origin issue. [#31273](https://github.com/ant-design/ant-design/pull/31273) [@Map1en](https://github.com/Map1en)
```

Comment:
Keep it as it is here.

Reply:
👍 Thank you. I want to take the time to thank you for assisting me with this PR. I really appreciate it

Reply:
> Please input PR changelog

I have no idea what to do, I would like to learn. This is my first PR to ant-design